_datascience/hutch_courses.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Fred Hutch Oriented DaSL Training
+primary_reviewers: vortexing
+---
+
+The Fred Hutch Data Science Lab's wider training efforts can be found on our [training page](https://hutchdatascience.org/training/).  However, in addition to data science training courses we create that are focused on a wider audience, we have also created a number of Fred Hutch oriented training materials.  
+
+
+## Cluster 101
+We collaborated with SciComp and created a course called [Cluster 101](https://leanpub.com/courses/fredhutch/fredhutchcluster101).  This course can be taken anytime, anywhere, for free (don't worry just set the LeanPub cost to zero) and it’ll help you check to make sure your account is set up to use the cluster and either help you check to make sure you understand the basics of how to use a cluster, or help you get up to speed.  It can even give you a certification for reporting between staff and lab leaders if need be.  
+
+If you take this course and want to give us feedback or would like to learn more about it, you can share your thoughts in [Slack in the #dasl-news channel](https://fhdata.slack.com/archives/C03T4QPNDA9). Or you can file an issue on [the course's GitHub repository](https://github.com/fhdsl/FH_Cluster_Guide).
```

Comment:
I should adjust these to be the #ask-dasl channel (link is here: https://fhdata.slack.com/archives/C043ZVCBY9Z)